tests/core/test_views_hello.py

```diff
@@ -78,7 +81,11 @@ def test_capabilities_can_be_specified_via_config(self):
 
         response = app.get('/')
         capabilities = response.json['capabilities']
-        expected = {"cook-coffee": {"description": "", "url": ""}}
+        expected = {
+            "cook-coffee": {"description": "", "url": ""},
+            "basicauth": {
+                'description': 'Very basic authentication sessions. Not for production use.',
+                'url': 'http://kinto.readthedocs.io/en/stable/api/1.x/authentication.html'}}
```

Comment:
I'm not super fan about the duplication of the basic auth capability here.

I guess we could just set `multiauth.policies` to an empty value for this test